contracts/cw4-group/src/contract.rs

```diff
@@ -173,8 +177,8 @@ fn query_total_weight(deps: Deps) -> StdResult<TotalWeightResponse> {
 fn query_member(deps: Deps, addr: String, height: Option<u64>) -> StdResult<MemberResponse> {
     let addr = deps.api.addr_validate(&addr)?;
     let weight = match height {
-        Some(h) => MEMBERS.may_load_at_height(deps.storage, &addr, h),
-        None => MEMBERS.may_load(deps.storage, &addr),
+        Some(h) => members().primary.may_load_at_height(deps.storage, &addr, h),
```

Comment:
Thing to expose in `IndexedSnapshotMap`: 
* `may_load_at_height`

Reply:
That should be enough for now.

In theory:
* `add_checkpoint`
* `remove_checkpoint`

would be good as well, but those are not used in any code yet (no `Strategy::Selected`)

Reply:
Implemented in #275.